packages/freezed_annotation/lib/freezed_annotation.dart

```diff
@@ -219,6 +219,7 @@ class Freezed {
     this.makeCollectionsUnmodifiable,
     this.addImplicitFinal = true,
     this.genericArgumentFactories = false,
+    this.finalize = false,
```

Comment:
We likely want to default to `null`, and pick up the default from `build.yaml`

Reply:
Will do that, but I wonder if there is an easy way to write a test that makes sure the value from the `build.yaml` is picked up correctly. Or is that something that can be generally considered as working and one doesn't need to care about it when just adding one more parameter?

Reply:
There's the integration folder for build.yaml stuff

Reply:
It is now nullable and gets its default value (`false`) from its `@JsonKey` annotation